samples/test-samples.cmake

```diff
@@ -38,9 +38,18 @@ foreach (SAMPLE data-sealing)
     COMMAND ${CMAKE_COMMAND} --build ${SOURCE_DIR}/${SAMPLE}
     WORKING_DIRECTORY ${SAMPLE_BUILD_DIR})
 
-  execute_process(
-    COMMAND ${CMAKE_COMMAND} --build ${SAMPLE_BUILD_DIR} --target run
-    RESULT_VARIABLE TEST_RESULT)
+  if (NOT DEFINED $ENV{OE_SIMULATION})
```

Comment:
Is this true if `OE_SIMULATION=0`? I think I've seen that variable used as a 1/0 flag.

Also, it would probably be a bit cleaner to simulation semantics via `-D` to this script instead of as an environment variable.

Reply:
Updated.